windows-installer/renew-up.ps1

```diff
@@ -0,0 +1,12 @@
+$taskName = "Certbot Renew Task"
+
+$exists = Get-ScheduledTask | Where-Object {$_.TaskName -like $taskName}
+if ($exists) {
+    Unregister-ScheduledTask -TaskName $taskName -Confirm:$false
+}
+
+$action = New-ScheduledTaskAction -Execute 'Powershell.exe' -Argument '-NoProfile -WindowStyle Hidden -Command "certbot renew"'
+$triggerAM = New-ScheduledTaskTrigger -Daily -At 12am
+$triggerPM = New-ScheduledTaskTrigger -Daily -At 12pm
+$principal = New-ScheduledTaskPrincipal -UserId $env:UserName -LogonType S4U -RunLevel Highest
```

Comment:
Yes, it is the user that installed Certbot.

In fact I see the two approaches on Windows: some programs register their task with System, others with the current user. As a system administrator, taking ownership on a Windows server, I appreciate to know from which predecessor a given scheduled task comes from.

On the other end, since Certbot requires administrator privileges, removing the privileges to the user associated with the task will silently break the `certbot renew` process, since the task will not be able to gain privileges anymore.

So, for the sake of automation, let's move to `System` ...

Reply:
Done.

Reply:
1. Yes, it seems to work on various machine. In particular my own, configured in french, where the "System" account is named "Système" in the UI. I put it in capital letters to keep the idea it is like a constant.

2. Indeed! Done.